### DISCUSSION TITLE
Workflow is failing if no job can be ran due to condition

### DISCUSSION THREAD

**Discussion by didrocks** [community/community#25146] [Actions]
<p>I’m trying to avoid one job to run conditionnally. For instance, let’s say I want to control a “/rebase” command to only trigger the rebase job steps on PR:</p>
<pre><code>on:
  issue_comment:
    types: [created]

name: PR commands
jobs:
  rebase:
    name: Rebase
    if: github.event.issue.pull_request != '' &amp;&amp; contains(github.event.comment.body, '/rebase')
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
    - name: Automatic Rebase
      uses: cirrus-actions/rebase@1.2
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
</code></pre>
<p>The issue is that the workflow will exit in error (and so, send failure emails). In the Action UI, clicking on it will redirect to the previous running workflow of another type.</p>
<p>One workaround right now is to let the job starting, and duplicate the condition on each step:</p>
<pre><code>on:
  issue_comment:
    types: [created]

name: PR commands
jobs:
  rebase:
    name: Rebase
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@master
      if: github.event.issue.pull_request != '' &amp;&amp; contains(github.event.comment.body, '/rebase')
    - name: Automatic Rebase
      uses: cirrus-actions/rebase@1.2
      if: github.event.issue.pull_request != '' &amp;&amp; contains(github.event.comment.body, '/rebase')
      env:
        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
</code></pre>
<p>Is there any way to completely prevent the job to start, and thus, avoiding the cirrus-actions/rebase@1.2 docker image action build?</p>

**Comment by BrightRan** [comment, marked as answer]
<p>Hi <a class="mention" href="/didrocks">@didrocks</a> ,</p>
<p>In the past, if all jobs in a workflow were skipped with the conditions, the workflow will display as “skipped/no status” status with a grey icon. But now, it displays as failed status.</p>
<p>I had reported this issue to the appropriate engineering team, and the team has began to investigate it.<br>
Currently, we don’t have any other ways to skip the jobs. However, as a workaround, you can add a job to always run in the workflow, you do not need to add any usable steps in this job, and just execute a line of script that don’t make sense. At this situation, when all other jobs are skipped with the if condition, the workflow will not be failed status.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/6/61e4617dc98899db11eb55ac97b5878887f3d0b0.png" data-download-href="/uploads/short-url/dXZCkJcgR6nt7L892s1nEdcD6xO.png?dl=1" title="workaround.PNG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/46307996/181089141-344c60d1-301a-44b7-bf85-3f25ceead602.png" alt="workaround.PNG" data-base62-sha1="dXZCkJcgR6nt7L892s1nEdcD6xO" width="690" height="366" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/6/61e4617dc98899db11eb55ac97b5878887f3d0b0_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">workaround.PNG</span><span class="informations">754×400 11.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by tianhaoz95** [comment]
<p>What is the status on this? Is it by design or a bug?</p>
